src/XIVLauncher.Core/Style/StyleModelV1.cs

```diff
@@ -114,6 +114,15 @@ private StyleModelV1()
             { "NavWindowingHighlight", new Vector4(1, 1, 1, 0.7f) },
             { "NavWindowingDimBg", new Vector4(0.8f, 0.8f, 0.8f, 0.2f) },
             { "ModalWindowDimBg", new Vector4(0.8f, 0.8f, 0.8f, 0.35f) },
+            { "InputTextCursor", new Vector4(1, 1, 1, 1) },
+            { "TabSelected", new Vector4(0.08f, 0.08f, 0.08f, 0.94f) },
```

Comment:
This will change the selected tab color to blue, to match the old UI.
```suggestion
            { "TabSelected", ImGuiColors.Blue },
```

There's also several lines (97-99) that are no longer used. TabActive has been replaced by TabSelected (as above), and the TabUnfocused* keys have been replaced with TabDimmed*. So they can be deleted.